drivers/storage/rackspace/executor/rackspace_executor.go

```diff
@@ -0,0 +1,354 @@
+package executor
+
+// // StorageExecutor is the part of a storage driver that is downloaded at
+// // runtime by the libStorage client.
+// type StorageExecutor interface {
+// 	Driver
+// 	StorageExecutorFunctions
+// }
+
+// // StorageExecutorFunctions is the collection of functions that are required
+// // of a StorageExecutor.
+// type StorageExecutorFunctions interface {
+// 	// InstanceID returns the local system's InstanceID.
+// 	InstanceID(
+// 		ctx Context,
+// 		opts Store) (*InstanceID, error)
+
+// 	// NextDevice returns the next available device.
+// 	NextDevice(
+// 		ctx Context,
+// 		opts Store) (string, error)
+
+// 	// LocalDevices returns a map of the system's local devices.
+// 	LocalDevices(
+// 		ctx Context,
+// 		opts *LocalDevicesOpts) (*LocalDevices, error)
+// }
+
+import (
+	//"bufio"
+	//"fmt"
+	//"io"
+	"io/ioutil"
+	//"os"
+	"os/exec"
+//	"path/filepath"
+	"regexp"
+	"strings"
+	
+	"github.com/akutz/gofig"
+	"github.com/akutz/goof"
+
+	"github.com/emccode/libstorage/api/registry"
+	"github.com/emccode/libstorage/api/types"
+	"github.com/emccode/libstorage/drivers/storage/rackspace"
+
+)
+
+// driver is the storage executor for the VFS storage driver.
+type driver struct {
+	config gofig.Config
+}
+
+// const (
```

Comment:
Hi @akiraw,

Please remove this unused constant or justify why it is commented out and still left in the file. Thanks!